packages/shorebird_code_push_protocol/lib/src/models/release.dart

```diff
@@ -8,7 +9,7 @@ part 'release.g.dart';
 /// A release can have zero or more patches applied to it.
 /// {@endtemplate}
 @JsonSerializable()
-class Release {
+class Release extends Equatable {
```

Comment:
Should it also be `@immutable`?  Typically equatable is only "safe" if also immutable.